### ISSUE TITLE
save broken on master with Docile master

### ISSUE THREAD

**Issue by tshort** [MichaelHatherly/Lexicon.jl#24]
I'm not sure if Lexicon and Docile are always supposed to be in sync in master, but save is broken with changes in Docile:

```julia
julia> save("t.html", Lexicon)
INFO: writing documentation to t.html
ERROR: `writemime` has no method matching writemime(::IOStream, ::MIME{symbol("text/html")}, ::Array{ASCIIString,1})
 in writemime at /home/tshort/.julia/v0.3/Lexicon/src/render/html.jl:46
 in anonymous at /home/tshort/.julia/v0.3/Lexicon/src/render/html.jl:14
 in open at iostream.jl:137
 in save at /home/tshort/.julia/v0.3/Lexicon/src/render/html.jl:12
```
I'd been using the last tagged version.

**Comment by MichaelHatherly** [comment]
Yes, the master branches aren't in sync unfortunately. Some of the major changes in Docile are causing this. I'm in the midst of rewriting the `query` code for Lexicon, once that's done I'll be sorting out the problems with `save`.

**Comment by MichaelHatherly** [comment]
This is most likely fixed with 0f60c19538b87887b23d728e33e3e776f66b85ac, though some formatting changes in the resulting output may have slipped in. Was caused by changing Docile to only store the file names of manual entries (ie. ``@docstrings(manual = ["foobar.md"])``) rather than the actual content of the file. Doing it this way improves package loads times slightly.
